astropy/coordinates/matching.py

```diff
@@ -301,7 +308,8 @@ def search_around_sky(coords1, coords2, seplimit, storekdtree="kdtree_sky"):
         The second set of coordinates, which will be searched for matches from
         ``coords1`` within ``seplimit``. Cannot be a scalar coordinate.
     seplimit : `~astropy.units.Quantity` ['angle']
-        The on-sky separation to search within.
+        The on-sky separation to search within. It should be a single separation or
+        have the same shape than ``coords1``.
```

Comment:
"have the same shape *as* ``coords1``" - this also in the other docstring and in the changelog entry.

Reply:
It should say that `seplimit` must be broadcastable to the same shape as `coords1`. That way if multidimensional coordinates will be handled properly we will not have to edit the description of `seplimit`.

Reply:
Done! :slightly_smiling_face: